microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -482,3 +482,10 @@ def _trigger_kdump_on_specified_cpu(
                 "The cpu count can't meet the test case's requirement. "
                 f"Expected more than {cpu_num} cpus, actual {cpu_count}"
             )
+
+    def after_case(self, log: Logger, **kwargs: Any) -> None:
+        # kdump cases will trigger crash
+        # therefore we mark the node dirty to prevent future testing on this environment
+        # to aviod detecting the panic call trace wrongly
+        node = kwargs["node"]
+        node.mark_dirty()
```

Comment:
As is, there is no reason to assign this to a variable.
```suggestion
        kwargs["node"].mark_dirty()
```

Is there any case there would be no node keyword?

Reply:
I think there is the node keyword for every case.